scrapy/utils/display.py

```diff
@@ -5,7 +5,7 @@
 import ctypes
 import platform
 import sys
-from distutils.version import LooseVersion as parse_version
+from packaging.version import Version as parse_version
```

Comment:
I think it'd be nice to make `packaging` an explicit dependency in setup.py (add it to install_requires), even if it's usually available. distutils is an stdlib module.

Reply:
Updated